tests/tasks/lib/test_sst.py

```diff
@@ -0,0 +1,149 @@
+    },
+    {
+        "guid": "train-4",
+        "text": "on the worst revenge-of-the-nerds clich\u00e9s the filmmakers could dredge up ",
+        "label": "0",
+    },
+]
+
+TOKENIZED_TRAIN_EXAMPLES = [
+    {
+        "guid": "train-0",
+        "text": ["hide", "new", "secretions", "from", "the", "parental", "units"],
+        "label_id": 0,
+    },
+    {
+        "guid": "train-1",
+        "text": ["contains", "no", "wit", ",", "only", "labored", "gags"],
+        "label_id": 0,
+    },
+    {
+        "guid": "train-2",
+        "text": [
+            "that",
+            "loves",
+            "its",
+            "characters",
+            "and",
+            "communicates",
+            "something",
+            "rather",
+            "beautiful",
+            "about",
+            "human",
+            "nature",
+        ],
+        "label_id": 1,
+    },
+    {
+        "guid": "train-3",
+        "text": ["remains", "utterly", "satisfied", "to", "remain", "the", "same", "throughout"],
+        "label_id": 0,
+    },
+    {
+        "guid": "train-4",
+        "text": [
+            "on",
+            "the",
+            "worst",
+            "revenge-of-the-nerds",
+            "clich\u00e9s",
+            "the",
+            "filmmakers",
+            "could",
+            "dredge",
+            "up",
+        ],
+        "label_id": 0,
+    },
+]
+
+
+FEATURIZED_TRAIN_EXAMPLE_0 = {
+    "guid": "train-0",
+    "input_ids": np.array([1, 4, 5, 6, 7, 8, 9, 10, 2, 0]),
+    "input_mask": np.array([1, 1, 1, 1, 1, 1, 1, 1, 1, 0]),
+    "segment_ids": np.array([0, 0, 0, 0, 0, 0, 0, 0, 0, 0]),
+    "label_id": 0,
+    "tokens": ["<cls>", "hide", "new", "secretions", "from", "the", "parental", "units", "<sep>"],
+}
+
+
+def test_sst_create_examples():
+    task = create_task_from_config_path(
+        os.path.join(os.path.dirname(__file__), "resources/sst.json"), verbose=False
+    )
+    examples = task.get_train_examples()
+    for example_dataclass, raw_example_dict in zip(examples, TRAIN_EXAMPLES):
+        assert example_dataclass.to_dict() == raw_example_dict
+
+
+def test_sst_create_tokenized_examples():
+    task = create_task_from_config_path(
+        os.path.join(os.path.dirname(__file__), "resources/sst.json"), verbose=False
+    )
+    examples = task.get_train_examples()
+    # the dummy tokenizer requires a vocab — using a Counter here to find that vocab from the data:
+    token_counter = Counter()
+    for example in examples:
+        token_counter.update(example.text.split())
+    token_vocab = list(token_counter.keys())
+    tokenizer = SimpleSpaceTokenizer(vocabulary=token_vocab)
+    tokenized_examples = [example.tokenize(tokenizer) for example in examples]
+    for tokenized_example, expected_tokenized_example in zip(
+        tokenized_examples, TOKENIZED_TRAIN_EXAMPLES
+    ):
+        assert tokenized_example.to_dict() == expected_tokenized_example
+
+
+def test_sst_create_featurized_example():
```

Comment:
similar to question above, curious if it would be acceptable to move all `assert()` to this function and remove the above tests.

Reply:
👍 , https://github.com/jiant-dev/jiant/pull/34#discussion_r434865270